tests/Backend/CommonPart1/TableWithConfigurationLoadTest.php

```diff
@@ -0,0 +1,175 @@
+        "description": ""
+      },
+      {
+        "sql": "CREATE TABLE {{ id(destSchemaName) }}.{{ id(destTableName ~ rand ~ '_tmp') }} WITH (DISTRIBUTION=ROUND_ROBIN,CLUSTERED COLUMNSTORE INDEX) AS SELECT a.[id],a.[NAME] FROM (SELECT COALESCE([id], '') AS [id],COALESCE([NAME], '') AS [NAME], ROW_NUMBER() OVER (PARTITION BY [id] ORDER BY [id]) AS \"_row_number_\" FROM {{ id(stageSchemaName) }}.{{ id(stageTableName) }}) AS a WHERE a.\"_row_number_\" = 1",
+        "description": ""
+      },
+      {
+        "sql": "RENAME OBJECT {{ id(destSchemaName) }}.{{ id(destTableName) }} TO {{ id(destTableName ~ rand ~ '_tmp_rename') }}",
+        "description": ""
+      },
+      {
+        "sql": "RENAME OBJECT {{ id(destSchemaName) }}.{{ id(destTableName ~ rand ~ '_tmp') }} TO {{ id(destTableName) }}",
+        "description": ""
+      },
+      {
+        "sql": "DROP TABLE {{ id(destSchemaName) }}.{{ id(destTableName ~ rand ~ '_tmp_rename') }}",
+        "description": ""
+      },
+      {
+        "sql": "IF OBJECT_ID (N'{{ id(destSchemaName) }}.{{ id(destTableName ~ rand ~ '_tmp') }}', N'U') IS NOT NULL DROP TABLE {{ id(destSchemaName) }}.{{ id(destTableName ~ rand ~ '_tmp') }}",
+        "description": ""
+      },
+      {
+        "sql": "IF OBJECT_ID (N'{{ id(destSchemaName) }}.{{ id(destTableName ~ rand ~ '_tmp_rename') }}', N'U') IS NOT NULL DROP TABLE {{ id(destSchemaName) }}.{{ id(destTableName ~ rand ~ '_tmp_rename') }}",
+        "description": ""
+      }
+    ]
+  }
+}
+JSON;
+        $jsonDecoded = json_decode(
+            $json,
+            true,
+            512,
+            JSON_THROW_ON_ERROR | JSON_PRETTY_PRINT
+        );
+
+        $queriesOverride = [];
+        $queriesOverride['ingestionFullLoad'] = $jsonDecoded['output'];
+
+        // HTML NOWDOC used so that autoformat does not reformat SQL queries inside the strings
+        $tableId = $this->prepareTableWithConfiguration($tableName, [
+                'migrations' => [
+                    [
+                        'sql' => /** @lang TSQL */ <<<SQL
+CREATE TABLE {{ id(bucketName) }}.{{ id(tableName) }} ([id] INTEGER, [NAME] VARCHAR(100))
+SQL,
+                        'description' => 'first ever',
+                    ],
+                ],
+                'queriesOverride' => $queriesOverride,
+            ]
+        );
+
+        $csvFile = new CsvFile(__DIR__ . '/../../_data/languages.csv');
+        $fileId = $this->_client->uploadFile(
+            $csvFile->getPathname(),
+            (new FileUploadOptions())
+                ->setNotify(false)
+                ->setIsPublic(false)
+                ->setCompress(true)
+                ->setTags(['table-import'])
+        );
+
+        $this->_client->writeTableAsyncDirect($tableId, [
+            'dataFileId' => $fileId,
+        ]);
```

Comment:
tady to pada na:
```
    [ * message] => Execution of custom table query failed because of : An exception occurred while executing a query: SQLSTATE[42000]: [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]Not able to validate external location because The remote server returned an error: (403) Forbidden. (executed query: COPY INTO [MJ_KBC_201-in_c-API-tests-9ee1a5d0d4bf09ae665601ba5d75adc67e0a633f].[#__temp_csvimport635fc7ab3af606_47144961]
FROM 'https://kbcfsvefph4ilrbqde.blob.core.windows.net/exp-15-files-201-13273-2022-10-31/languages.csv.gz'
```

Reply:
je to tak neměls nastavené managed identity, nastavil sem ti to.

Reply:
jo vidis, jsme se o tom bavili a nakonec nedotahli... dik 🍻

Reply:
![image](https://user-images.githubusercontent.com/180148/199182138-2a6ff6b2-75e6-4f50-8021-7ebc5f6f7afb.png)

Reply:
No, me to ted pada na nenalezene evente 🤷 
```
Caused by
 GuzzleHttp\Exception\ClientException: Client error: `GET http://connection-apache/v2/storage/events/20059489` resulted in a `404 Not Found` response:
 {"error":"Event not found","exceptionId":"connection-devel-4d8a8a6292881d8441e3f224ddf1385e","code":"storage.events.notFound"}
```

Reply:
✅ uz mi to proslo.